src/Files.App/Helpers/CloudDrivesDetector.cs

```diff
@@ -235,5 +236,36 @@ private static Task<IEnumerable<ICloudProvider>> DetectpCloudDrive()
 
 			return Task.FromResult<IEnumerable<ICloudProvider>>(results);
 		}
+
+		private static Task<IEnumerable<ICloudProvider>> DetectNutstoreDrive()
+		{
+			var results = new List<ICloudProvider>();
+			using var NutstoreKey = Registry.CurrentUser.OpenSubKey(@"SOFTWARE\Nutstore");
+
+			if (NutstoreKey is not null)
+			{
+				string iconPath = Path.Combine(Environment.GetEnvironmentVariable("ProgramFiles"), "Nutstore", "Nutstore.exe");
```

Comment:
It would be a good idea to create a constant for `Environment.GetEnvironmentVariable("ProgramFiles")`.

Reply:
Should I do it in Constants.cs?

Reply:
Probably best to do it here